bootcamp/javascript/conditions/05-if-statement/task/05-if-statement.md

```diff
@@ -0,0 +1,36 @@
+## Exercise 1
+
+T = Temperature
+D = Day
+W = Weather
+
+Close the window if it's `Monday` and the temperature is below `22`.
+
+
+## Exercise 2
+
+T = Temperature
+D = Day
+W = Weather
+
+Close the window if it's `Monday` and the temperature is below `22`.
+Close the window if it's `Tuesday` and the temperature is below `24`.
+
+
+## Exercise 3
+
+T = Temperature
+D = Day
+W = Weather
+
+Close the window if it's not `Wednesday` and the temperature is below `22`.
+
+
+## Exercise 4
+
+T = Temperature
+D = Day
+W = Weather
+
+Close the window if it's `Monday` or `Tuesday` and the temperature is below `22`.
+Close the window any day if the weather is `Rain`
```

Comment:
This topic contains a task and solution folder, but there is no html, js or css file present in the task folder, please add something